config/environments/docker/production.ini

```diff
@@ -101,7 +101,7 @@ ckan.search.solr_commit = true
 #       Add ``datapusher`` to enable DataPusher
 #		Add ``resource_proxy`` to enable resorce proxying and get around the
 #		same origin policy
-ckan.plugins = stats text_view image_view recline_view geodatagov datagov_harvest ckan_harvester geodatagov_geoportal_harvester z3950_harvester arcgis_harvester waf_harvester_collection geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_metadata spatial_query resource_proxy spatial_harvest_metadata_api recline_preview datagovtheme datajson_harvest datajson archiver qa googleanalyticsbasic extlink report broken_link_report
+ckan.plugins = stats text_view image_view recline_view geodatagov datagov_harvest ckan_harvester geodatagov_geoportal_harvester z3950_harvester arcgis_harvester waf_harvester_collection geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_metadata spatial_query resource_proxy spatial_harvest_metadata_api recline_preview datagovtheme datajson_harvest datajson archiver qa googleanalyticsbasic extlink report broken_link_report saml2
```

Comment:
We don't want to enabled saml2 in development unless there is a way to still login with username/password. We don't have a way to setup saml2 authentication for development right now.